cmd/arena/commands/resource.go

```diff
@@ -77,3 +77,71 @@ func gpuInContainerDeprecated(container v1.Container) int64 {
 
 	return val.Value()
 }
+
+// filter out the pods with resources
+func resourcePods(pods []v1.Pod) (podsWithResource []v1.Pod) {
+	for _, pod := range pods {
+		if gpuInPod(pod) > 0 || cpuInPod(pod) > 0 || memoryInPod(pod) > 0 {
+			podsWithResource = append(podsWithResource, pod)
+		}
+	}
+	return podsWithResource
+}
+
+// The way to get CPU Count of Node
+func cpuInNode(node v1.Node) int64 {
+	val, ok := node.Status.Capacity[v1.ResourceCPU]
+	if !ok {
+		return 0
+	}
+
+	return val.Value()
+}
+
+func cpuInPod(pod v1.Pod) (cpuCount int64) {
+	containers := pod.Spec.Containers
+	for _, container := range containers {
+		cpuCount += cpuInContainer(container)
+	}
+
+	return cpuCount
+}
+
+func cpuInContainer(container v1.Container) int64 {
+	val, ok := container.Resources.Limits[v1.ResourceCPU]
```

Comment:
use container.Resources.Requests[v1.ResourceCPU], not Limits. 

Please also check memoryInContainer.

Reply:
Yes, we should fix this, thanks.